externs/browser/html5.js

```diff
@@ -4286,3 +4315,64 @@ Plugin.prototype.length;
 
 /** @type {string} */
 Plugin.prototype.name;
+
+/**
+ * @see https://w3c.github.io/webcomponents/spec/custom/#custom-elements-api
+ * @constructor
+ */
+function CustomElementsRegistry() {}
```

Comment:
The name is `CustomElementRegistry`. Singular `Element`. https://html.spec.whatwg.org/multipage/scripting.html#custom-elements-api